<!DOCTYPE html><!--[if lt IE 7]> <html lang="en" class="ie ie6 lte9 lte8 lte7 os-mac"> <![endif]--><!--[if IE 7]> <html lang="en" class="ie ie7 lte9 lte8 lte7 os-mac"> <![endif]--><!--[if IE 8]> <html lang="en" class="ie ie8 lte9 lte8 os-mac"> <![endif]--><!--[if IE 9]> <html lang="en" class="ie ie9 lte9 os-mac"> <![endif]--><!--[if gt IE 9]> <html lang="en" class="os-mac"> <![endif]--><!--[if !IE]><!--><html xmlns="http://www.w3.org/1999/xhtml" lang="en" class="os-mac"><!--<![endif]--><head>
<meta charset="UTF-8" />
<title>Milos Pjanic
| LinkedIn
</title>
<meta name="referrer" content="origin" />
<meta http-equiv="X-UA-Compatible" content="IE=edge" />
<meta name="pageImpressionID" content="2f1576f9-30f8-41be-975b-3512b4653b1e" />
<meta name="appName" content="chrome" />
<meta name="pageKey" content="public_profile_v3_desktop" />
<meta name="treeID" content="dKZWP4majBSAN2u8XysAAA==" />
<meta name="globalTrackingUrl" content="//www.linkedin.com/mob/tracking" />
<meta name="globalTrackingAppName" content="chrome" />
<meta name="globalTrackingAppId" content="webTracking" />
<!--[if lte IE 8]>
  <link rel="shortcut icon" href="https://static.licdn.com/scds/common/u/images/logos/favicons/v1/16x16/favicon.ico">
<![endif]-->
<!--[if IE 9]>
  <link rel="shortcut icon" href="https://static.licdn.com/scds/common/u/images/logos/favicons/v1/favicon.ico">
<![endif]-->
<link rel="icon" href="https://static.licdn.com/scds/common/u/images/logos/favicons/v1/favicon.ico" />
<link rel="apple-touch-icon-precomposed" href="https://static.licdn.com/scds/common/u/img/icon/apple-touch-icon.png" />
<meta name="msapplication-TileImage" content="https://static.licdn.com/scds/common/u/images/logos/linkedin/logo-in-win8-tile-144_v1.png" />
<meta name="msapplication-TileColor" content="#0077B5" />
<meta name="application-name" content="LinkedIn" />
<meta content="https://static.licdn.com/scds/concat/common/js?v=0.1.186" name="RemoteNavJSContentBaseURL" />
<script type="text/javascript" src="https://static.licdn.com/scds/concat/common/js?h=7ndrn0f9fw0hum7uoqcjcnzne-95d8d303rtd0n9wj4dcjbnh2c-7vr4nuab43rzvy2pgq7yvvxjk-9qa4rfxekcw3lt2c06h7p0kmf"></script>
<link rel="stylesheet" type="text/css" href="https://static.licdn.com/scds/concat/common/css?h=dof06vs7ja0lriiyk6dewc5gg-a4kjc5uqttio53azw54aex6s3-as8kt5bqspxc01tl9cizqa37j" />
<script></script>
<!--[if IE 8]><script src="https://static.licdn.com/scds/common/u/lib/polyfill/1.0.2/ie8-polyfill.min.js"></script><![endif]-->
<!--[if IE 9]><script src="https://static.licdn.com/scds/common/u/lib/polyfill/1.0.2/ie9-polyfill.min.js"></script><![endif]-->
<meta name="lnkd-track-json-lib" content="https://static.licdn.com/scds/concat/common/js?h=2jds9coeh4w78ed9wblscv68v-ebbt2vixcc5qz0otts5io08xv" />


<meta id="jet-chromeVersion" name="jet-chromeVersion" content="0.0.2000-RC8.59472" />
<meta id="jet-scHost" name="jet-scHost" content="static.licdn.com" />
<meta id="jet-csrfToken" name="jet-csrfToken" content="ajax:4419914781333978430" />
<script type="text/javascript" src="https://static.licdn.com/scds/concat/common/js?h=e2lgukqldpqool72t8g7tysag-3nuvxgwg15rbghxm1gpzfbya2-1nm61x5u7981e88m10hpaekkm-mv3v66b8q0h1hvgvd3yfjv5f-14k913qahq3mh0ac0lh0twk9v-28vfzntn604yf2k0jo32aleiv-5gw567fe2s0ma0alr8a0m8236"></script>
<link rel="canonical" href="https://www.linkedin.com/in/milos-pjanic-a3b39746" /><link rel="stylesheet" href="https://static.licdn.com/sc/h/3drnstogfpa81ujm9ugh09gxg" /><script type="text/javascript" src="https://static.licdn.com/sc/h/4tcd0mh70bs89zecpaumh27p1"></script><meta http-equiv="content-type" content="text/html; charset=UTF-8" /><meta name="robots" content="noarchive" /><meta name="description" content="View Milos Pjanic’s professional profile on LinkedIn. LinkedIn is the world's largest business network, helping professionals like Milos Pjanic discover inside connections to recommended job candidates, industry experts, and business partners." /><meta name="og:title" content="Milos Pjanic | LinkedIn" /><meta name="og:description" content="View Milos Pjanic’s professional profile on LinkedIn. LinkedIn is the world's largest business network, helping professionals like Milos Pjanic discover inside connections to recommended job candidates, industry experts, and business partners." /><meta property="og:image" content="https://media.licdn.com/mpr/mpr/shrinknp_200_200/p/7/000/262/1d2/1818a78.jpg" /><link rel="alternate" hreflang="en" href="https://www.linkedin.com/in/milos-pjanic-a3b39746" /><link rel="alternate" hreflang="de" href="https://www.linkedin.com/in/milos-pjanic-a3b39746/de" />
<link rel="stylesheet" type="text/css" href="https://static.licdn.com/sc/p/com.linkedin.public-profile-frontend%3Apublic-profile-frontend-static-content%2B0.0.666/f/%2Fpublic-profile-frontend%2Ffonts%2Fsource-sans-pro-font.css" /></head>
<body class="js guest advocate-modal-visible" dir="ltr" id="pagekey-public_profile_v3_desktop">
<div id="application-body">
<header id="layout-header" class="layout-header-or-footer" role="banner">
<div id="header-banner">
<div class="wrapper">
<a class="banner-logo-container guest" href="http://www.linkedin.com/">
<h2 id="li-logo" class="logo">
LinkedIn Home
</h2>
</a>
<nav role="navigation" class="nav guest-nav" aria-label="Main site navigation">
<ul role="menubar">
<li role="menuitem" class="nav-item">
<a class="nav-link" href="http://www.linkedin.com/static?key=what_is_linkedin&amp;trk=hb_what">
What is LinkedIn?
</a>
</li>
<li role="menuitem" class="nav-item">
<a rel="nofollow" class="nav-link" href="/start/join?trk=hb_join">
Join Today
</a>
</li>
<li role="menuitem" class="nav-item">
<a rel="nofollow" class="nav-link" href="https://www.linkedin.com/uas/login?goback=&amp;trk=hb_signin">
Sign In
</a>
</li>
</ul>
</nav>
</div>
</div>
<div class="a11y-content">
<a id="a11y-content-link" tabindex="0" name="a11y-content">Main content starts below.</a>
</div>
<script data-page-js-type="lix">(function(n,r,a){r=window[n]=window[n]||{};r['jsecure_injectAlert']='enabled';r['jsecure_Dialog']='control';}('__li__lix_registry__'));</script>
<div id="global-alert-queue" class="layout-wrapper"></div></header>
<main id="layout-main" role="main">
<div id="advocate-modal" class="show-reg show"><div class="modal-va-fix"><div class="modal-content"><div class="flipcard"><section class="register-form"><div class="modal-header"><h2 class="modal-headline">Join now to see <strong>Milos’s full profile. </strong>It's free!</h2></div><div class="modal-body"><form id="advocate-reg-form" action="https://www.linkedin.com/start/view-full-profile-prefill?_ed=0_3AyDXbdm3z1XO_NPPHT6udWTYe7jO6pEyPuGrmIDyu7xRsoBHEEX3zS0BJbp1Wpk" method="POST" data-jsenabled="check"><div class="input-fields"><label for="jw-first-name">First name</label><input id="jw-first-name" name="firstName" type="text" /><label for="jw-last-name">Last name</label><input id="jw-last-name" name="lastName" type="text" /><label for="jw-email-or-phone">Email</label><input id="jw-email-or-phone" name="emailAddress" type="email" /><label for="jw-password">Password (6 or more characters)</label><input autocomplete="new-password" id="jw-password" name="password" type="password" /></div><div class="form-errors"></div><div class="join-disclaimer">By clicking Join now, you agree to the LinkedIn <a class="legal-link" href="https://www.linkedin.com/legal/user-agreement">User Agreement</a>, <a class="legal-link" href="https://www.linkedin.com/legal/privacy-policy">Privacy Policy</a>, and <a class="legal-link" href="https://www.linkedin.com/legal/cookie-policy">Cookie Policy</a>.</div><button class="join-btn" type="submit">Join now</button><p class="reg-option">or</p><button type="button" class="fb-btn lazy-loaded" data-genie-url="https://www.linkedin.com/genie/startauth?pop=TRUE&amp;url=%2Fstart%2Fthird-party-callback%3Fprovider%3DFACEBOOK&amp;sec=TRUE&amp;as=urn%3Ali%3AgaapScript%3Aprofile%2Efacebook"><p>Continue with Facebook</p></button></form><div class="alt-action login">Already on LinkedIn? <a href="https://www.linkedin.com/uas/login?session_redirect=https%3A%2F%2Fwww%2Elinkedin%2Ecom%2Fin%2Fmilos-pjanic-a3b39746&amp;amp;fromSignIn=true&amp;amp;trk=public_profile_joinwall-login" class="toggle-link">Sign in</a></div></div></section><section class="login-form"><div class="modal-header"><h2 class="modal-headline">Sign in to see <strong>Milos’s full profile. </strong>It's free!</h2></div><div class="modal-body"><form id="advocate-login-form" action="https://www.linkedin.com/uas/login-submit" method="POST"><div class="input-fields"><input id="lw-email-or-phone" name="session_key" type="text" placeholder="Email" /><input id="lw-password" name="session_password" type="password" placeholder="Password" /></div><div class="form-errors"></div><button class="login-btn" type="submit">Sign in</button><div class="forgot-password"><a class="forgot-password-link" href="https://www.linkedin.com/uas/request-password-reset">Forgot password?</a></div><input name="loginCsrfParam" id="loginCsrfParam-login" type="hidden" value="c3a53dff-af97-4037-84ee-a76eaa52f858" /><input name="sourceAlias" id="sourceAlias-login" type="hidden" value="0_7r5yezRXCiA_H0CRD8sf6DhOjTKUNps5xGTqeX8EEoi" /></form></div><div class="modal-footer"><div class="alt-action sign-up">Don't have an account? <a href="https://www.linkedin.com/start/view-full-profile?_ed=0_3AyDXbdm3z1XO_NPPHT6udWTYe7jO6pEyPuGrmIDyu7xRsoBHEEX3zS0BJbp1Wpk&amp;amp;trk=public_profile_loginwall-join" class="toggle-link">Join now</a></div></div></section></div></div></div></div><script src="https://static.licdn.com/sc/h/45xj22djwratc92bmcasdu83t"></script><code id="isMobile" style="display: none;"><!--false--></code><code id="pageKey" style="display: none;"><!--"public_profile_v3_desktop"--></code><code id="jsBeaconUrl" style="display: none;"><!--"https://www.linkedin.com/analytics/noauthtracker"--></code><code id="currentPageUrl" style="display: none;"><!--"https://www.linkedin.com/in/milos-pjanic-a3b39746"--></code><code id="isCookiedMember" style="display: none;"><!--false--></code><div id="wrapper" class="advocate-modal-visible"><div id="profile"><section id="topcard" class="profile-section"><div class="profile-card vcard"><div class="profile-picture" data-section="picture"><a class="photo" href="https://media.licdn.com/mpr/mpr/shrinknp_200_200/p/7/000/262/1d2/1818a78.jpg"></a><a href="https://www.linkedin.com/start/view-full-profile?_ed=0_3AyDXbdm3z1XO_NPPHT6udWTYe7jO6pEyPuGrmIDyu7xRsoBHEEX3zS0BJbp1Wpk&amp;trk=pprof-0-ts-view_full-0"><img lazyload="true" class="image lazy-loaded" alt="Milos Pjanic" width="200" height="200" src="https://media.licdn.com/mpr/mpr/shrinknp_200_200/p/7/000/262/1d2/1818a78.jpg" /></a></div><div class="profile-overview"><div class="profile-overview-content"><h1 id="name" class="fn">Milos Pjanic</h1><p class="headline title" data-section="headline">Postdoctoral fellow at Stanford University School of Medicine</p><dl id="demographics"><dt>Location</dt><dd class="descriptor adr"><span class="locality">Stanford, California</span></dd><dt>Industry</dt><dd class="descriptor">Biotechnology</dd></dl><table class="extra-info"><tbody><tr data-section="currentPositionsDetails"><th>Current</th><td><ol><li><span class="org"><a href="https://www.linkedin.com/company/stanford-university-school-of-medicine?trk=ppro_cprof">Stanford University School of Medicine</a></span></li></ol></td></tr><tr data-section="pastPositionsDetails"><th>Previous</th><td><ol><li>Institute of Molecular genetics and Genetic Engineering, </li><li><a href="https://www.linkedin.com/company/center-for-genomic-regulation-crg-?trk=ppro_cprof">Center for Genomic Regulation (CRG)</a>, </li><li><a href="https://www.linkedin.com/company/lunduniversity?trk=ppro_cprof">Lund University</a></li></ol></td></tr><tr data-section="educationsDetails"><th>Education</th><td><ol><li>University of Lausanne</li></ol></td></tr><tr class="websites" data-section="websites"><th>Websites</th><td><ol><li><a href="https://www.linkedin.com/redir/redirect?url=http%3A%2F%2Fwww%2Emilospjanic%2Eblogspot%2Ecom&amp;urlhash=cW7c&amp;trk=ppro_website">Milos Pjanic | blog</a></li></ol></td></tr></tbody></table><div class="member-connections"><strong>172</strong> connections</div></div></div></div><div class="reg-upsell  advocate-cta-enabled lazy-loaded"><h2>View Milos’ full profile. It's free!</h2><h3>Your colleagues, classmates, and 400 million other professionals are on LinkedIn.</h3><a href="https://www.linkedin.com/start/view-full-profile?_ed=0_3AyDXbdm3z1XO_NPPHT6udWTYe7jO6pEyPuGrmIDyu7xRsoBHEEX3zS0BJbp1Wpk&amp;trk=public_profile_advocate-cta-view-full" class="signup-button">View Milos’ Full Profile</a></div></section><section class="profile-section single-section" data-section="posts" id="feed"><h3 class="title">Milos’ Activity</h3><ul class="activities"><li class="activity"><a href="https://www.linkedin.com/start/join?session_redirect=https%3A%2F%2Fwww%2Elinkedin%2Ecom%2Fhp%2Fupdate%2F6196854456349376512%3FpathWildcard%3D6196854456349376512&amp;trk=public_profile_feed"><div class="image-container"><img class="lazy-load" data-delayed-url="https://media.licdn.com/media-proxy/ext?w=800&amp;h=800&amp;hash=rikroEnXJFiqDou5tvoDSQkI6NM%3D&amp;ora=1%2CaFBCTXdkRmpGL2lvQUFBPQ%2CxAVta9Er0VOmnBxUkE8g7qqMqAGz60oIUpKRAW__H3D0vsrFICu2KJOcZrChpVsJeSwe6XlFEOuoSTn8EJO8Y4jvfcIry8A" /></div><div class="content"><h5 class="item-title">Hacking Away at Health and Ancestry</h5><h6 class="item-subtitle">Milos liked</h6></div></a></li></ul></section><section id="summary" data-section="summary" class="profile-section"><h3 class="title">Summary</h3><div class="description"><p>My major fields of interest are computational biology and bioinformatics, coupled with the passion for the next-generation sequencing technologies, and a profound scientific interest in genomics, transcriptomics, regulation of gene expression, specificity of binding of transcription factors to the genome, histone modifications, nucleosome positioning, long-range genomic interactions and compartmentalization of the genome. My research lies on the frontier of the contemporary computational genomics, with the emphasis on development and testing of scripts and algorithms for the analysis of human genome and transcriptome. My focus is the improvement of methods for the various applications of the next generation sequencing, such as chromatin - immunoprecipitation sequencing or ChIP-Seq, RNA-sequencing or RNA-Seq, and probing open chromatin, DNase-Seq/ATAC-Seq, in order to answer key biological question that will ultimately help us understand better the underlying mechanisms of life. As a postdoc at Stanford’s Cardiovascular Institute, I am elucidating complex networks of interactions of transcription factors in human cardiac and vascular tissues, and molecular mechanisms that explain how cardiovascular disease risk-associated genomic loci confer disease risk. I am also employing allele specific computational pipelines to the existing next generation sequencing techniques, i.e. ChIP-Seq and RNA-Seq, in combination with the generation of eQTL data for human arterial smooth muscle cells (primary cell type of atherosclerotic lesions) to identify the causal variants that underlie disease susceptibility. In addition, I am modeling vascular SMC tissue-specific open chromatin with ATAC-Seq and DNase-Seq to understand the underlying mechanisms for cardiovascular disease causal variants. I am also active as a blogger, started a blog www.genomicscode.org and continually post UNIX and R related tips and resolve computational problems that can be applied to genomics.<br /></p></div></section><section id="experience" class="profile-section"><h3 class="title">Experience</h3><ul class="positions"><li class="position" data-section="currentPositionsDetails"><header><h5 class="logo"><a href="https://www.linkedin.com/company/stanford-university-school-of-medicine?trk=ppro_cprof"><img data-delayed-url="https://media.licdn.com/media/AAEAAQAAAAAAAAdDAAAAJDcxNTYwZjQwLWEzYzItNDNhYy1hYWM4LWMzMDM4Njg3YmM2Mw.png" class="lazy-load" alt="Stanford University School of Medicine" /></a></h5><h4 class="item-title"><a href="https://www.linkedin.com/title/postdoctoral-research-fellow?trk=pprofile_title">Postdoctoral Research Fellow</a></h4><h5 class="item-subtitle"><a href="https://www.linkedin.com/company/stanford-university-school-of-medicine?trk=ppro_cprof">Stanford University School of Medicine</a></h5></header><div class="meta"><span class="date-range"><time>March 2014</time> – Present (2 years 10 months)</span></div><p class="description" data-section="currentPositions">Stanford, California 94305, USA<br /></p></li><li class="position" data-section="pastPositionsDetails"><header><h4 class="item-title"><a href="https://www.linkedin.com/title/bioinformatics-analyst?trk=pprofile_title">Bioinformatics Analyst</a></h4><h5 class="item-subtitle">Institute of Molecular genetics and Genetic Engineering</h5></header><div class="meta"><span class="date-range"><time>2013</time> – <time>2014</time> (1 year)</span><span class="location">Serbia</span></div></li><li class="position" data-section="pastPositionsDetails"><header><h5 class="logo"><a href="https://www.linkedin.com/company/center-for-genomic-regulation-crg-?trk=ppro_cprof"><img data-delayed-url="https://media.licdn.com/media/p/3/005/055/19e/1c7e371.png" class="lazy-load" alt="Center for Genomic Regulation (CRG)" /></a></h5><h4 class="item-title"><a href="https://www.linkedin.com/title/bioinformatician?trk=pprofile_title">Bioinformatician</a></h4><h5 class="item-subtitle"><a href="https://www.linkedin.com/company/center-for-genomic-regulation-crg-?trk=ppro_cprof">Center for Genomic Regulation (CRG)</a></h5></header><div class="meta"><span class="date-range"><time>2012</time> – <time>2013</time> (1 year)</span><span class="location">Barcelona Area, Spain</span></div><p class="description" data-section="pastPositions">Bioinformatician</p></li><li class="position" data-section="pastPositionsDetails"><header><h5 class="logo"><a href="https://www.linkedin.com/company/lunduniversity?trk=ppro_cprof"><img data-delayed-url="https://media.licdn.com/media/AAEAAQAAAAAAAAgSAAAAJDc4YzBkOWRkLWVjNTMtNDQ0ZC1iMWFhLWVjZDlhMjcwYjQ4ZQ.png" class="lazy-load" alt="Lund University" /></a></h5><h4 class="item-title"><a href="https://www.linkedin.com/title/visiting-researcher?trk=pprofile_title">Visiting researcher</a></h4><h5 class="item-subtitle"><a href="https://www.linkedin.com/company/lunduniversity?trk=ppro_cprof">Lund University</a></h5></header><div class="meta"><span class="date-range"><time>2011</time> – <time>2012</time> (1 year)</span><span class="location">Sweden</span></div></li><li class="position" data-section="pastPositionsDetails"><header><h4 class="item-title"><a href="https://www.linkedin.com/title/graduate-teaching-assistant?trk=pprofile_title">Graduate Teaching Assistant</a></h4><h5 class="item-subtitle">École Polytechnique Fédérale de Lausanne</h5></header><div class="meta"><span class="date-range"><time>2006</time> – <time>2010</time> (4 years)</span><span class="location">Switzerland</span></div></li></ul></section><section id="courses" data-section="courses" class="profile-section"><h3 class="title">Courses</h3><ul><li><h4 class="item-title">University of Lausanne</h4><div class="courses-list-container description"><ul class="courses-list"><li class="course"><span class="translated translation" data-field-name="Course.Name">English (Council of Europe Level C1 - equivalent to Certificate in Advanced English CAE, obtained from Centre de langues, University of Lausanne, Switzerland)</span><span class="original translation" data-field-name="Course.Name">English (Council of Europe Level C1 - equivalent to Certificate in Advanced English CAE, obtained from Centre de langues, University of Lausanne, Switzerland)</span></li><li class="course"><span class="translated translation" data-field-name="Course.Name">French (Council of Europe Level A2, obtained from Centre de langues, University of Lausanne, Switzerland)</span><span class="original translation" data-field-name="Course.Name">French (Council of Europe Level A2, obtained from Centre de langues, University of Lausanne, Switzerland)</span></li></ul></div></li></ul></section><section id="languages" data-section="languages" class="profile-section"><h3 class="title">Languages</h3><ul><li class="language"><div class="wrap"><h4 class="name">English</h4><p class="proficiency">Full professional proficiency</p></div></li><li class="language"><div class="wrap"><h4 class="name">French</h4><p class="proficiency">Elementary proficiency</p></div></li></ul></section><section id="awards" data-section="awards" class="profile-section"><h3 class="title">Honors &amp; Awards</h3><ul><li class="award"><header><h4 class="item-title">University Award</h4><h5 class="item-subtitle">University of Belgrade</h5></header><div class="meta"><span class="date-range"><time>2000</time></span></div><p class="description">2000<br />University of Belgrade, University Award<br />For the work performed in a student research project: “BIOFORM”</p></li><li class="award"><header><h4 class="item-title">Outstanding Research Poster Award</h4><h5 class="item-subtitle">Stanford-Karolinska Cardiovascular Research Symposium 2016</h5></header><div class="meta"><span class="date-range"><time>October 2016</time></span></div><p class="description">Stanford University School of Medicine - Stanford University Cardiovascular Institute,<br />Outstanding Research Poster Award, Stanford-Karolinska Cardiovascular Research Symposium 2016 - In recognition of your work and research in cardiovascular medicine.<br /></p></li><li class="award"><header><h4 class="item-title">Reviewers'​ Choice Abstract</h4><h5 class="item-subtitle">. American Society of Human Genetics</h5></header><div class="meta"><span class="date-range"><time>October 2016</time></span></div><p class="description">Reviewers' Choice Abstract Award for top abstracts. American Society of Human Genetics. Vancouver, 2016.</p></li></ul></section><section id="skills" data-section="skills" class="profile-section skills-section"><h3 class="title">Skills</h3><ul class="pills"><input type="checkbox" id="skills-expander-state-checkbox" class="expander-state-checkbox" /><li class="skill"><a href="https://www.linkedin.com/topic/molecular-biology?trk=pprofile_topic" title="Molecular Biology"><span class="wrap">Molecular Biology</span></a></li><li class="skill"><a href="https://www.linkedin.com/topic/bioinformatics?trk=pprofile_topic" title="Bioinformatics"><span class="wrap">Bioinformatics</span></a></li><li class="skill"><a href="https://www.linkedin.com/topic/cell-culture?trk=pprofile_topic" title="Cell Culture"><span class="wrap">Cell Culture</span></a></li><li class="skill"><a href="https://www.linkedin.com/topic/cell-biology?trk=pprofile_topic" title="Cell Biology"><span class="wrap">Cell Biology</span></a></li><li class="skill"><a href="https://www.linkedin.com/topic/qpcr?trk=pprofile_topic" title="qPCR"><span class="wrap">qPCR</span></a></li><li class="skill"><a href="https://www.linkedin.com/topic/biotechnology?trk=pprofile_topic" title="Biotechnology"><span class="wrap">Biotechnology</span></a></li><li class="skill"><a href="https://www.linkedin.com/topic/genomics?trk=pprofile_topic" title="Genomics"><span class="wrap">Genomics</span></a></li><li class="skill"><a href="https://www.linkedin.com/topic/protein-expression?trk=pprofile_topic" title="Protein Expression"><span class="wrap">Protein Expression</span></a></li><li class="skill"><a href="https://www.linkedin.com/topic/gene-expression?trk=pprofile_topic" title="Gene Expression"><span class="wrap">Gene Expression</span></a></li><li class="skill"><a href="https://www.linkedin.com/topic/r?trk=pprofile_topic" title="R"><span class="wrap">R</span></a></li><li class="skill"><a href="https://www.linkedin.com/topic/cell?trk=pprofile_topic" title="Cell"><span class="wrap">Cell</span></a></li><li class="skill"><a href="https://www.linkedin.com/topic/sequencing?trk=pprofile_topic" title="Sequencing"><span class="wrap">Sequencing</span></a></li><li class="skill"><a href="https://www.linkedin.com/topic/biochemistry?trk=pprofile_topic" title="Biochemistry"><span class="wrap">Biochemistry</span></a></li></ul><div class="translation-voter-wrap"><div class="translation-voter"><span>How's this translation?</span><ul class="voter-form"><li class="action option positive"><span class="action positive">Great</span></li><li class="separator">•</li><li class="action option negative"><span class="action negative">Has errors</span></li></ul></div><div class="translation-voter-response">Thanks for your help!</div></div></section><section id="publications" data-section="publications" class="profile-section"><h3 class="title">Publications</h3><ul><li class="publication"><header><h4 class="item-title"><a href="https://www.linkedin.com/redir/redirect?url=http%3A%2F%2Fwww%2Ebiomedcentral%2Ecom%2F1471-2164%2F12%2F181%2Fabstract&amp;urlhash=9vKd&amp;trk=prof-publication-title-link" title="Nuclear factor I revealed as family of promoter binding transcription activators." class="external-link">Nuclear factor I revealed as family of promoter binding transcription activators.</a></h4><h5 class="item-subtitle">BMC Genomics</h5><div class="meta"><span class="date-range"><time>2011</time></span></div></header><p class="description">Abstract<br />Background<br />Multiplex experimental assays coupled to computational predictions are being increasingly employed for the simultaneous analysis of many specimens at the genome scale, which quickly generates very large amounts of data. However, inferring valuable biological information from the comparisons of very large genomic datasets still represents an enormous challenge.<br /><br />Results<br />As a study model, we chose the NFI/CTF family of mammalian transcription factors and we compared the results obtained from a genome-wide study of its binding sites with chromatin structure assays, gene expression microarray data, and in silico binding site predictions. We found that NFI/CTF family members preferentially bind their DNA target sites when they are located around transcription start sites when compared to control datasets generated from the random subsampling of the complete set of NFI binding sites. NFI proteins preferably associate with the upstream regions of genes that are highly expressed and that are enriched in active chromatin modifications such as H3K4me3 and H3K36me3. We postulate that this is a causal association and that NFI proteins mainly act as activators of transcription. This was documented for one member of the family (NFI-C), which revealed as a more potent gene activator than repressor in global gene expression analysis. Interestingly, we also discovered the association of NFI with the tri-methylation of lysine 9 of histone H3, a chromatin marker previously associated with the protection against silencing of telomeric genes by NFI.<br /><br />Conclusion<br />Taken together, we illustrate approaches that can be taken to analyze large genomic data, and provide evidence that NFI family members may act in conjunction with specific chromatin modifications to activate gene expression.</p><dl class="contributors"><dt>Authors: </dt><dd><ul><li class="contributor"><a href="https://www.linkedin.com/in/milos-pjanic-a3b39746">Milos Pjanic</a>, </li><li class="contributor">Petar Pjanic, </li><li class="contributor">Christoph Schmid, </li><li class="contributor">Giovanna Ambrosini, </li><li class="contributor"><a href="https://www.linkedin.com/in/armelle-gaussin-8220a415">Armelle Gaussin</a>, </li><li class="contributor">Genta Plasari, </li><li class="contributor">Christian Mazza, </li><li class="contributor">Philipp Bucher, </li><li class="contributor">Nicolas Mermod</li></ul></dd></dl></li><li class="publication"><header><h4 class="item-title"><a href="https://www.linkedin.com/redir/redirect?url=http%3A%2F%2Fwww%2Ebiomedcentral%2Ecom%2F1471-2164%2F14%2F99%2Fabstract&amp;urlhash=3sGk&amp;trk=prof-publication-title-link" title="Nuclear Factor I genomic binding associates with chromatin boundaries" class="external-link">Nuclear Factor I genomic binding associates with chromatin boundaries</a></h4><h5 class="item-subtitle">BMC Genomics</h5><div class="meta"><span class="date-range"><time>February 2013</time></span></div></header><p class="description">Abstract<br />Background<br /><br />The Nuclear Factor I (NFI) family of DNA binding proteins (also called CCAAT box transcription factors or CTF) is involved in both DNA replication and gene expression regulation. Using chromatin immuno-precipitation and high throughput sequencing (ChIP-Seq), we performed a genome-wide mapping of NFI DNA binding sites in primary mouse embryonic fibroblasts.<br />Results<br /><br />We found that in vivo and in vitro NFI DNA binding specificities are indistinguishable, as in vivo ChIP-Seq NFI binding sites matched predictions based on previously established position weight matrix models of its in vitro binding specificity. Combining ChIP-Seq with mRNA profiling data, we found that NFI preferentially associates with highly expressed genes that it up-regulates, while binding sites were under-represented at expressed but unregulated genes. Genomic binding also correlated with markers of transcribed genes such as histone modifications H3K4me3 and H3K36me3, even outside of annotated transcribed loci, implying NFI in the control of the deposition of these modifications. Positional correlation between + and - strand ChIP-Seq tags revealed that, in contrast to other transcription factors, NFI associates with a nucleosomal length of cleavage-resistant DNA, suggesting an interaction with positioned nucleosomes. In addition, NFI binding prominently occurred at boundaries displaying discontinuities in histone modifications specific of expressed and silent chromatin, such as loci submitted to parental allele-specific imprinted expression.<br />Conclusions<br /><br />Our data thus suggest that NFI nucleosomal interaction may contribute to the partitioning of distinct chromatin domains and to epigenetic gene expression regulation. </p><dl class="contributors"><dt>Authors: </dt><dd><ul><li class="contributor"><a href="https://www.linkedin.com/in/milos-pjanic-a3b39746">Milos Pjanic</a>, </li><li class="contributor">Christoph D Schmid, </li><li class="contributor"><a href="https://www.linkedin.com/in/armelle-gaussin-8220a415">Armelle Gaussin</a>, </li><li class="contributor">Giovanna Ambrosini, </li><li class="contributor">Jozef Adamcik, </li><li class="contributor">Petar Pjanic, </li><li class="contributor">Genta Plasari, </li><li class="contributor">Jan Kerschgens, </li><li class="contributor">Giovani Dietler, </li><li class="contributor">Philipp Bucher, </li><li class="contributor">Nicolas Mermod</li></ul></dd></dl></li><li class="publication"><header><h4 class="item-title">Molecular characterization of a human matrix attachment region epigenetic regulator</h4><h5 class="item-subtitle">PLOS ONE</h5><div class="meta"><span class="date-range"><time>November 2013</time></span></div></header><p class="description">Matrix attachment regions (MAR) generally act as epigenetic regulatory sequences that increase gene expression, and they were proposed to partition chromosomes into loop-forming domains. However, their molecular mode of action remains poorly understood. Here, we assessed the possible contribution of the AT-rich core and adjacent transcription factor binding motifs to the transcription augmenting and anti-silencing effects of human MAR 1-68. Either flanking sequences together with the AT-rich core were required to obtain the full MAR effects. Shortened MAR derivatives retaining full MAR activity were constructed from combinations of the AT-rich sequence and multimerized transcription factor binding motifs, implying that both transcription factors and the AT-rich microsatellite sequence are required to mediate the MAR effect. Genomic analysis indicated that MAR AT-rich cores may be depleted of histones and enriched in RNA polymerase II, providing a molecular interpretation of their chromatin domain insulator and transcriptional augmentation activities.</p><dl class="contributors"><dt>Authors: </dt><dd><ul><li class="contributor">Salina Arope, </li><li class="contributor"><a href="https://www.linkedin.com/in/niamh-harraghy-561b2313">Niamh Harraghy</a>, </li><li class="contributor"><a href="https://www.linkedin.com/in/milos-pjanic-a3b39746">Milos Pjanic</a>, </li><li class="contributor">Nicolas Mermod</li></ul></dd></dl></li><li class="publication"><header><h4 class="item-title"><a href="https://www.linkedin.com/redir/redirect?url=http%3A%2F%2Fwww%2Encbi%2Enlm%2Enih%2Egov%2Fpubmed%2F26020946&amp;urlhash=7bmD&amp;trk=prof-publication-title-link" title="Coronary Artery Disease Associated Transcription Factor TCF21 Regulates Smooth Muscle Precursor Cells that Contribute to the Fibrous Cap" class="external-link">Coronary Artery Disease Associated Transcription Factor TCF21 Regulates Smooth Muscle Precursor Cells that Contribute to the Fibrous Cap</a></h4><h5 class="item-subtitle">Plos Genetics</h5><div class="meta"><span class="date-range"><time>May 2015</time></span></div></header><p class="description">Abstract<br />Recent genome wide association studies have identified a number of genes that contribute to the risk for coronary heart disease. One such gene, TCF21, encodes a basic-helix-loop-helix transcription factor believed to serve a critical role in the development of epicardial progenitor cells that give rise to coronary artery smooth muscle cells (SMC) and cardiac fibroblasts. Using reporter gene and immunolocalization studies with mouse and human tissues we have found that vascular TCF21 expression in the adult is restricted primarily to adventitial cells associated with coronary arteries and also medial SMC in the proximal aorta of mouse. Genome wide RNA-Seq studies in human coronary artery SMC (HCASMC) with siRNA knockdown found a number of putative TCF21 downstream pathways identified by enrichment of terms related to CAD, including "vascular disease," "disorder of artery," and "occlusion of artery," as well as disease-related cellular functions including "cellular movement" and "cellular growth and proliferation." In vitro studies in HCASMC demonstrated that TCF21 expression promotes proliferation and migration and inhibits SMC lineage marker expression. Detailed in situ expression studies with reporter gene and lineage tracing revealed that vascular wall cells expressing Tcf21 before disease initiation migrate into vascular lesions of ApoE-/- and Ldlr-/- mice. While Tcf21 lineage traced cells are distributed throughout the early lesions, in mature lesions they contribute to the formation of a subcapsular layer of cells, and others become associated with the fibrous cap. The lineage traced fibrous cap cells activate expression of SMC markers and growth factor receptor genes. </p><dl class="contributors"><dt>Authors: </dt><dd><ul><li class="contributor"><a href="https://www.linkedin.com/in/milos-pjanic-a3b39746">Milos Pjanic</a></li></ul></dd></dl></li><li class="publication"><header><h4 class="item-title"><a href="https://www.linkedin.com/redir/redirect?url=http%3A%2F%2Fjournals%2Eplos%2Eorg%2Fplosgenetics%2Farticle%3Fid%3D10%2E1371%2Fjournal%2Epgen%2E1005202&amp;urlhash=CIV6&amp;trk=prof-publication-title-link" title="Characterization of TCF21 downstream target regions identifies a transcriptional network linking multiple independent coronary artery disease loci" class="external-link">Characterization of TCF21 downstream target regions identifies a transcriptional network linking multiple independent coronary artery disease loci</a></h4><h5 class="item-subtitle">Plos Genetics</h5><div class="meta"><span class="date-range"><time>May 2015</time></span></div></header><p class="description">Abstract<br /><br />To functionally link coronary artery disease (CAD) causal genes identified by genome wide association studies (GWAS), and to investigate the cellular and molecular mechanisms of atherosclerosis, we have used chromatin immunoprecipitation sequencing (ChIP-Seq) with the CAD associated transcription factor TCF21 in human coronary artery smooth muscle cells (HCASMC). Analysis of identified TCF21 target genes for enrichment of molecular and cellular annotation terms identified processes relevant to CAD pathophysiology, including “growth factor binding,” “matrix interaction,” and “smooth muscle contraction.” We characterized the canonical binding sequence for TCF21 as CAGCTG, identified AP-1 binding sites in TCF21 peaks, and by conducting ChIP-Seq for JUN and JUND in HCASMC confirmed that there is significant overlap between TCF21 and AP-1 binding loci in this cell type. Expression quantitative trait variation mapped to target genes of TCF21 was significantly enriched among variants with low P-values in the GWAS analyses, suggesting a possible functional interaction between TCF21 binding and causal variants in other CAD disease loci. Separate enrichment analyses found over-representation of TCF21 target genes among CAD associated genes, and linkage disequilibrium between TCF21 peak variation and that found in GWAS loci, consistent with the hypothesis that TCF21 may affect disease risk through interaction with other disease associated loci. Interestingly, enrichment for TCF21 target genes was also found among other genome wide association phenotypes, including height and inflammatory bowel disease, suggesting a functional profile important for basic cellular processes in non-vascular tissues. Thus, data and analyses presented here suggest that study of GWAS transcription factors may be a highly useful approach to identifying disease gene interactions and thus pathways that may be relevant to complex disease etiology.</p><dl class="contributors"><dt>Authors: </dt><dd><ul><li class="contributor"><a href="https://www.linkedin.com/in/milos-pjanic-a3b39746">Milos Pjanic</a></li></ul></dd></dl></li><li class="publication"><header><h4 class="item-title">Molecular basis of regulatory variation at coronary heart disease associated loci</h4><h5 class="item-subtitle">Atherosclerosis 241 (1), e17DOI: 10.1016/j.atherosclerosis.2015.04.077</h5><div class="meta"><span class="date-range"><time>April 2015</time></span></div></header><p class="description">Molecular basis of regulatory variation at coronary heart disease associated loci<br />C Miller, M Pjanic, TL Assimes, SB Montgomery, WJ Greenleaf, T Quertermous<br />Atherosclerosis 241 (1), e17DOI: 10.1016/j.atherosclerosis.2015.04.077 </p><dl class="contributors"><dt>Authors: </dt><dd><ul><li class="contributor"><a href="https://www.linkedin.com/in/milos-pjanic-a3b39746">Milos Pjanic</a></li></ul></dd></dl></li><li class="publication"><header><h4 class="item-title"><a href="https://www.linkedin.com/redir/redirect?url=https%3A%2F%2Fdoc%2Erero%2Ech%2Frecord%2F22788%2Ffiles%2Fmaz_nfr_sm1%2Epdf&amp;urlhash=DCB_&amp;trk=prof-publication-title-link" title="Random sampling algorithm" class="external-link">Random sampling algorithm</a></h4><h5 class="item-subtitle">Rero Doc Digital library</h5></header><dl class="contributors"><dt>Authors: </dt><dd><ul><li class="contributor"><a href="https://www.linkedin.com/in/milos-pjanic-a3b39746">Milos Pjanic</a></li></ul></dd></dl></li><li class="publication"><header><h4 class="item-title"><a href="https://www.linkedin.com/redir/redirect?url=http%3A%2F%2Fatvb%2Eahajournals%2Eorg%2Fcontent%2F35%2F10%2F2079%2Efull&amp;urlhash=T-I0&amp;trk=prof-publication-title-link" title="From Locus Association to Mechanism of Gene Causality. The Devil Is in the Details" class="external-link">From Locus Association to Mechanism of Gene Causality. The Devil Is in the Details</a></h4><h5 class="item-subtitle">Arteriosclerosis, Thrombosis, and Vascular Biology</h5><div class="meta"><span class="date-range"><time>September 2015</time></span></div></header><p class="description">Arguably, one of the greatest advances in biomedical research in the past decades has been the association of human allelic variation with complex human diseases and phenotypes. Certainly, initially these findings were met with skepticism, given the underpowered and misleading number of candidate gene association studies that preceded the genome-wide association (GWA) era. Also, there was considerable incredulity when it became clear that identified variation in reasonably powered GWA studies (GWAS) could not address the bulk of attributable genetic risk for the majority of diseases. However, scientists in academia and industry are now increasingly recognizing the importance of studying these loci for better understanding of disease pathways and developing new therapeutics. This is perhaps most significant for atherosclerotic coronary artery disease (CAD), the primary source of mortality and morbidity worldwide, for which no single drug has yet been developed to target the primary disease process in the vessel wall.<br /></p><dl class="contributors"><dt>Authors: </dt><dd><ul><li class="contributor"><a href="https://www.linkedin.com/in/milos-pjanic-a3b39746">Milos Pjanic</a>, </li><li class="contributor"><a href="https://www.linkedin.com/in/clint-miller-b6b51122">Clint Miller</a>, </li><li class="contributor"><a href="https://www.linkedin.com/in/thomas-quertermous-08bb4024">Thomas Quertermous</a></li></ul></dd></dl></li><li class="publication"><header><h4 class="item-title"><a href="https://www.linkedin.com/redir/redirect?url=http%3A%2F%2Ff1000research%2Ecom%2Farticles%2F4-1349%2Fv1&amp;urlhash=PYnX&amp;trk=prof-publication-title-link" title="Transcription factor regulation as a mechanism of confounding effects between distinct human traits" class="external-link">Transcription factor regulation as a mechanism of confounding effects between distinct human traits</a></h4><h5 class="item-subtitle">F1000Research</h5><div class="meta"><span class="date-range"><time>November 2015</time></span></div></header><p class="description">Genome-wide association studies (GWAS) to date have discovered thousands of genetic variants linked to human diseases and traits, which hold the potential to unravel the mechanisms of complex phenotypes. However, given that the majority of these associated variants reside in non-coding genomic regions, their predicted cis and trans-regulatory functions remain largely undefined. Here we show that correlation between human diseases and traits can follow geographical distribution of human populations, and that the underlying mechanism is at least partly genetically based. We report two Type 2 Diabetes (T2D) GWAS variants (rs7903146 and rs12255372) in the TCF7L2 locus that regulate expression in skin tissues but not lymphoblastoid or adipose tissues, of the KITLG gene that encodes an important regulator of melanogenesis and light hair color in European populations. We also report extensive binding events of TCF7L2 protein in the promoter region, immediate upstream region and first intron of the KITLG gene, which supports a trans-interaction between TCF7L2 and KITLG. We further show that both light hair color and T2D genetic variants are correlated with geographic latitude. Taken together, our observations suggest that natural variation in transcription factor loci in European human populations may be an underlying and confounding factor for the geographical correlation between human phenotypes, such as type 2 diabetes and light hair color. We postulate that transcription factor regulation may confound the correlation between seemingly diverse human traits. Furthermore, our findings demonstrate the importance of dissecting the genomic architecture of GWAS loci using multiple genetic and genomic datasets.</p><dl class="contributors"><dt>Authors: </dt><dd><ul><li class="contributor"><a href="https://www.linkedin.com/in/milos-pjanic-a3b39746">Milos Pjanic</a>, </li><li class="contributor"><a href="https://www.linkedin.com/in/clint-miller-b6b51122">Clint Miller</a>, </li><li class="contributor"><a href="https://www.linkedin.com/in/thomas-quertermous-08bb4024">Thomas Quertermous</a></li></ul></dd></dl></li><li class="publication"><header><h4 class="item-title">TCF21 Regulates Coronary Artery Disease Causing Aryl-hydrocarbon Receptor Gene Expression and its Downstream Pathway Activation by Environmental Ligands</h4><h5 class="item-subtitle">Circulation</h5><div class="meta"><span class="date-range"><time>January 2015</time></span></div></header><p class="description">Circulation 01/2015; 132(Suppl 3):A17671. </p><dl class="contributors"><dt>Authors: </dt><dd><ul><li class="contributor">Juyong B Kim, </li><li class="contributor"><a href="https://www.linkedin.com/in/milos-pjanic-a3b39746">Milos Pjanic</a>, </li><li class="contributor">Olga Sazanova, </li><li class="contributor">Tina Wang, </li><li class="contributor">Clint Miller, </li><li class="contributor">Thomas Quertermous</li></ul></dd></dl></li><li class="publication"><header><h4 class="item-title"><a href="https://www.linkedin.com/redir/redirect?url=https%3A%2F%2Fgithub%2Ecom%2Fmilospjanic%2Ffisher-test-for-genomic-overlaps&amp;urlhash=blQw&amp;trk=prof-publication-title-link" title="Fisher statistical test for overlaps of two sets of genomic regions in a given genomic background" class="external-link">Fisher statistical test for overlaps of two sets of genomic regions in a given genomic background</a></h4><h5 class="item-subtitle">GitHub</h5><div class="meta"><span class="date-range"><time>April 2016</time></span></div></header><p class="description">04/2016, Languages: bash/R, Repository: https://github.com/milospjanic/fisher-test-for-genomic-overlaps, DOI: 10.13140/RG.2.1.2851.6249<br /><br />This is a combined bash/R script that will generate a Fisher test p-value that shows significance for the overlap of two sets of genomic regions (for example from ChIP-Seq experiments).<br /><br />To calculate Fisher p-value you would need three BED files: 1. First bed file to overlap, i.e. file1.bed 2. Second bed file for overlap, i.e. file2.bed 3. Bed file that will serve as genomic background for overlaps, i.e. background.bed<br /><br />Note that file names do not have to be file1.bed, file2.bed, background.bed, this is just an example.<br /><br />As genomic background you can use e.g. combined ENCODE set of open chromatin regions or a similar data set. Genomic background is necessary to calculate the constituents of the Fisher test contingency matrix: overlap of A and B in the background BG set, A but not B in BG, B but not A in BG, BG without A and B.<br /><br />We provide a combined ENCODE DHS data set in a bed file which you can use as a background set, background.bed.</p><dl class="contributors"><dt>Authors: </dt><dd><ul><li class="contributor"><a href="https://www.linkedin.com/in/milos-pjanic-a3b39746">Milos Pjanic</a></li></ul></dd></dl></li><li class="publication"><header><h4 class="item-title"><a href="https://www.linkedin.com/redir/redirect?url=https%3A%2F%2Fgithub%2Ecom%2Fmilospjanic%2Fhypergeometric-test-for-genomic-overlaps&amp;urlhash=A2gW&amp;trk=prof-publication-title-link" title="Hypergeometric statistical test for overlaps of two sets of genomic regions in a predefined genomic background" class="external-link">Hypergeometric statistical test for overlaps of two sets of genomic regions in a predefined genomic background</a></h4><h5 class="item-subtitle">GitHub</h5><div class="meta"><span class="date-range"><time>April 2016</time></span></div></header><p class="description">DOI: 10.13140/RG.2.1.4753.1926 <br />04/2016, Languages: bash, R, Repository: https://github.com/milospjanic/hypergeometric-test-for-genomic-overlaps, DOI: 10.13140/RG.2.1.4753.1926<br /><br />This is a combined bash/R script that will generate a hypergeometric p-value that shows significance for the overlap of two sets of genomic regions (for example from ChIP-Seq experiments).<br /><br />To calculate hypergeometric p-value you would need three BED files: 1. First bed file to overlap, i.e. file1.bed 2. Second bed file for overlap, i.e. file2.bed 3. Bed file that will serve as genomic background for overlaps, i.e. background.bed<br /><br />Note that file names are optional.<br /><br />As genomic background you can use e.g. combined ENCODE set of open chromatin regions or a similar data set. Genomic background is necessary to calculate the constituents of the hypergeometric test elements: A-B overlap in BG, A no B in BG, total BG minus BG-A overlap, B no A in BG.<br /><br />We provide a combined ENCODE DHS data set in a bed file which you can use as a background set, background.bed. </p><dl class="contributors"><dt>Authors: </dt><dd><ul><li class="contributor"><a href="https://www.linkedin.com/in/milos-pjanic-a3b39746">Milos Pjanic</a></li></ul></dd></dl></li><li class="publication"><header><h4 class="item-title"><a href="https://www.linkedin.com/redir/redirect?url=https%3A%2F%2Fgithub%2Ecom%2Fmilospjanic%2Fensemble2genename&amp;urlhash=6c7x&amp;trk=prof-publication-title-link" title="Append gene symbols to human ENSEMBLE gene IDs" class="external-link">Append gene symbols to human ENSEMBLE gene IDs</a></h4><h5 class="item-subtitle">GitHub</h5><div class="meta"><span class="date-range"><time>April 2016</time></span></div></header><p class="description">This is combined bash/R script that will use a file with human ENSEMBLE geneIDs in a first column of a file and append a gene name to it, while keeping the structure of the file from other columns. Ensemble2genename sets its host to ensembl.org thus it could be especially useful when biomaRt site is down.</p><dl class="contributors"><dt>Authors: </dt><dd><ul><li class="contributor"><a href="https://www.linkedin.com/in/milos-pjanic-a3b39746">Milos Pjanic</a></li></ul></dd></dl></li><li class="publication"><header><h4 class="item-title"><a href="https://www.linkedin.com/redir/redirect?url=https%3A%2F%2Fgithub%2Ecom%2Fmilospjanic%2Fbinomial-test-for-genomic-overlaps&amp;urlhash=l6sN&amp;trk=prof-publication-title-link" title="Binomial statistical test for overlaps of two sets of genomic regions" class="external-link">Binomial statistical test for overlaps of two sets of genomic regions</a></h4><h5 class="item-subtitle">GitHub</h5><div class="meta"><span class="date-range"><time>April 2016</time></span></div></header><p class="description">This is a combined bash/R script that will generate a binomial p-value that shows significance for the overlap of two sets of genomic regions (for example from ChIP-Seq experiments).<br />To calculate binomial p-value you would need two bed files as inputs. Note that binomial test does not require background set as do Fisher and hypergeometric tests.</p><dl class="contributors"><dt>Authors: </dt><dd><ul><li class="contributor"><a href="https://www.linkedin.com/in/milos-pjanic-a3b39746">Milos Pjanic</a></li></ul></dd></dl></li><li class="publication"><header><h4 class="item-title"><a href="https://www.linkedin.com/redir/redirect?url=https%3A%2F%2Fgithub%2Ecom%2Fmilospjanic%2Fbamsplit2bigwig&amp;urlhash=Tsbc&amp;trk=prof-publication-title-link" title="Bamsplit2bigwig, a tool to convert and separate bam files into strand specific bigwig for UCSC Genome Browser" class="external-link">Bamsplit2bigwig, a tool to convert and separate bam files into strand specific bigwig for UCSC Genome Browser</a></h4><h5 class="item-subtitle">GitHub</h5><div class="meta"><span class="date-range"><time>April 2016</time></span></div></header><p class="description">This script will take a bam file mapped to the human genome hg19 and convert it to two strand specific bigwig files that could be loaded to UCSC Genome Browser. It depends on bedtools bamtobed (aka bamToBed), and it will attempt to download and execute three UCSC scripts, bedItemOverlapCount, bedGraphToBigWig and fetchChromSizes, so make sure your connection is working. Converting bam to bigwig and separating according to strand could be useful to visualize nucleosomal or protein binding 'footprints' in ChIP-Seq experiments as at the boundaries of protein-DNA interactions will be marked with reads mapping on different DNA strands.</p><dl class="contributors"><dt>Authors: </dt><dd><ul><li class="contributor"><a href="https://www.linkedin.com/in/milos-pjanic-a3b39746">Milos Pjanic</a></li></ul></dd></dl></li><li class="publication"><header><h4 class="item-title"><a href="https://www.linkedin.com/redir/redirect?url=https%3A%2F%2Fgithub%2Ecom%2Fmilospjanic%2Fbam2bigwig&amp;urlhash=jgbP&amp;trk=prof-publication-title-link" title="Bam2bigwig, a tool to convert bam files into bigwig for UCSC Genome Browser" class="external-link">Bam2bigwig, a tool to convert bam files into bigwig for UCSC Genome Browser</a></h4><h5 class="item-subtitle">GitHub</h5><div class="meta"><span class="date-range"><time>April 2016</time></span></div></header><p class="description">This script will take a bam file mapped to the human genome hg19 and convert it to a bigwig file that could be loaded to UCSC Genome Browser. It depends on bedtools bamtobed (aka bamToBed), and it will attempt to download and execute three UCSC scripts, bedItemOverlapCount, bedGraphToBigWig and fetchChromSizes, so make sure your connection is working.<br /></p><dl class="contributors"><dt>Authors: </dt><dd><ul><li class="contributor"><a href="https://www.linkedin.com/in/milos-pjanic-a3b39746">Milos Pjanic</a></li></ul></dd></dl></li><li class="publication"><header><h4 class="item-title"><a href="https://www.linkedin.com/redir/redirect?url=https%3A%2F%2Fgithub%2Ecom%2Fmilospjanic%2FgwasCatalog2Bed2Category&amp;urlhash=MXTW&amp;trk=prof-publication-title-link" title="GwasCatalog2Bed2Category, tool to convert GWAS Catalog to phenotype specific bed files" class="external-link">GwasCatalog2Bed2Category, tool to convert GWAS Catalog to phenotype specific bed files</a></h4><h5 class="item-subtitle">GitHub</h5><div class="meta"><span class="date-range"><time>April 2016</time></span></div></header><p class="description">This is the script that will connect to http://www.genome.gov, download GWAS Catalog, convert it to a bed file with "chr position position+1 proxy_gene phenotype", and then take all your input search terms and select those entries in the catalog that match the term, and create separate bed files for each search term.<br /><br />Script will create and call two 'subscripts' GwasCatalog2Bed.sh and main.sh. Main.sh will be removed while GwasCatalog2Bed.sh will not be removed and you can use it afterwards to download whole GWAS Catalog in bed format.<br /><br />Files that will be generated are: GwasCatalog.bed, which contains shorter bed version of the entire GWAS Catalog and phenotype specific bed files.<br /><br />gwasCatalog2Bed2Category will also remove duplicates from phenotype/term specific bed files, that may occur as you may greb similar but different phenotypes that contain identical SNPs. Script will save those files as .bed.cut.sort.uniq and report the number of SNPs before and after removing deplicates.</p><dl class="contributors"><dt>Authors: </dt><dd><ul><li class="contributor"><a href="https://www.linkedin.com/in/milos-pjanic-a3b39746">Milos Pjanic</a></li></ul></dd></dl></li><li class="publication"><header><h4 class="item-title"><a href="https://www.linkedin.com/redir/redirect?url=https%3A%2F%2Fgithub%2Ecom%2Fmilospjanic%2Fbed2GwasCatalogBinomialMod1&amp;urlhash=j2B5&amp;trk=prof-publication-title-link" title="bed2GwasCatalogBinomialMod1, modified binomial test to apply in genomic analysis" class="external-link">bed2GwasCatalogBinomialMod1, modified binomial test to apply in genomic analysis</a></h4><h5 class="item-subtitle">GitHub</h5><div class="meta"><span class="date-range"><time>April 2016</time></span></div></header><p class="description">This script is a modification of the /bed2GwasCatalogBinomial and calculates binomial p-value for genomics overlaps using the following criteria. The P-values were computed using binomial cumulative distribution function b(x;n,p) in R (dbinom function). We set the parameter n equal to the total number of GWAS SNPs in a particular GWAS phenotype. Parameter x was set to the number of GWAS SNPs for a given GWAS phenotype that overlap input regions and parameter p was set to the fraction of the uniquely mappable human hg19 genome (calculated with subscript) that is localized in the input regions and contains assessed GWAS phenotype SNPs. Calculated binomial p-value equals the probability of having x or more of the n test genomic regions in the open chromatin domain given that the probability of that occurring for a single GWAS genomic location is p. Note the p-values may be inflated depending on your input files.<br /><br />This script will connect to GWAS Catalog and download the entire data set, create bed file, and parse and uniq according to the N-1 input arguments. Last argument provided to the bash script should be a bed file that will be used to intersect parsed bed files from the GWAS Catalog. Number of overlaps is reported and initial number of entries in parsed files. Finally, it will create an R script that will be executed to calculate binomial p-values for each overlap. Intermediary files will be removed, except: GwasCatalog.bed (entire catalog in a bed file), *gwascatalog.bed (parsed original files from GWAS Catalog), *gwascatalog.bed.cut.sort.uniq.chrXY (parsed and uniqed files from GWAS Catalog), *gwascatalog.bed.cut.sort.uniq.overlap (overlap with parsed files, GWAS SNP positions), *gwascatalog.bed.cut.sort.uniq.overlap.input.int.cut (overlap with parsed files, input regions), GwasCatalog2Bed.sh (sciprt to download GWAS Catalog and convert to bed). Note that we enabled case insensitive search in this script.</p><dl class="contributors"><dt>Authors: </dt><dd><ul><li class="contributor"><a href="https://www.linkedin.com/in/milos-pjanic-a3b39746">Milos Pjanic</a></li></ul></dd></dl></li><li class="publication"><header><h4 class="item-title"><a href="https://www.linkedin.com/redir/redirect?url=https%3A%2F%2Fgithub%2Ecom%2Fmilospjanic%2Fgwasanalytics&amp;urlhash=5M9g&amp;trk=prof-publication-title-link" title="Gwasanalytics, a collection of scripts to analyze ChiP-Seq/ATAC-Seq-GWAS correlations." class="external-link">Gwasanalytics, a collection of scripts to analyze ChiP-Seq/ATAC-Seq-GWAS correlations.</a></h4><h5 class="item-subtitle">GitHub</h5><div class="meta"><span class="date-range"><time>May 2016</time></span></div></header><p class="description">gwasanalytics<br />Gwas Analytics consists of a series of scripts that will analyze Gwas Catalog phenotypes and the input bed file from the ChIP-Seq experiment (or any other experiment that involves partitioning of the genome into compartments, e.g. ATAC-Seq, DNase-Seq etc).<br /><br />bed2GwasCatalogBinomialGgplot<br />This script will calculate binomial p-values and fold change of genomic overlaps between the input bed file and various GWAS categories. Usage is fairly simple and the output file is a publication ready pdf that is made in ggplot2 using the wesanderson color palette in R.</p><dl class="contributors"><dt>Authors: </dt><dd><ul><li class="contributor"><a href="https://www.linkedin.com/in/milos-pjanic-a3b39746">Milos Pjanic</a></li></ul></dd></dl></li><li class="publication"><header><h4 class="item-title"><a href="https://www.linkedin.com/redir/redirect?url=https%3A%2F%2Fgithub%2Ecom%2Fmilospjanic%2Fsam2BigWig&amp;urlhash=VQV7&amp;trk=prof-publication-title-link" title="Sam2BigWig, tool to convert sam files to bigwig for UCSC Genome Browser" class="external-link">Sam2BigWig, tool to convert sam files to bigwig for UCSC Genome Browser</a></h4><h5 class="item-subtitle">GitHub</h5><div class="meta"><span class="date-range"><time>May 2016</time></span></div></header><p class="description">sam2BigWig<br />This script will take a sam file mapped to the human genome hg19 and convert it to a bigwig file that could be loaded to UCSC Genome Browser. It depends on samtools, then bedtools bamtobed (aka bamToBed), and it will attempt to download and execute three UCSC scripts, bedItemOverlapCount, bedGraphToBigWig and fetchChromSizes, so make sure your connection is working.</p><dl class="contributors"><dt>Authors: </dt><dd><ul><li class="contributor"><a href="https://www.linkedin.com/in/milos-pjanic-a3b39746">Milos Pjanic</a></li></ul></dd></dl></li><li class="publication"><header><h4 class="item-title"><a href="https://www.linkedin.com/redir/redirect?url=https%3A%2F%2Fgithub%2Ecom%2Fmilospjanic%2FfileMulti2TableMod1&amp;urlhash=tcxd&amp;trk=prof-publication-title-link" title="fileMulti2TableMod1, script to combine RNA-Seq count files from multiple conditions into one table." class="external-link">fileMulti2TableMod1, script to combine RNA-Seq count files from multiple conditions into one table.</a></h4><h5 class="item-subtitle">GitHub</h5><div class="meta"><span class="date-range"><time>May 2016</time></span></div></header><p class="description">Lets say you have multiple files with the same first column and you want to merge them. Use this awk script fileMulti2TableMod1. All files will be compared to the rows from 1st column of the 1st file. if one of the files has a missing value(s) even though they contain the correct element in first column, rows with missing values are not excluded but missing values are set to 0. Rows with column 1 not present in column 1 of the first file will not be shown, as well as those rows from the first file that are missing in other files' first columns. Only those rows with 1st columns present in every file are shown.</p><dl class="contributors"><dt>Authors: </dt><dd><ul><li class="contributor"><a href="https://www.linkedin.com/in/milos-pjanic-a3b39746">Milos Pjanic</a></li></ul></dd></dl></li><li class="publication"><header><h4 class="item-title"><a href="https://www.linkedin.com/redir/redirect?url=http%3A%2F%2Fwww%2Enature%2Ecom%2Fncomms%2F2016%2F160708%2Fncomms12092%2Ffull%2Fncomms12092%2Ehtml&amp;urlhash=nEnP&amp;trk=prof-publication-title-link" title="Integrative functional genomics identifies regulatory mechanisms at coronary artery disease loci" class="external-link">Integrative functional genomics identifies regulatory mechanisms at coronary artery disease loci</a></h4><h5 class="item-subtitle">Nature Communications</h5><div class="meta"><span class="date-range"><time>July 2016</time></span></div></header><p class="description">Coronary artery disease (CAD) is the leading cause of mortality and morbidity, driven by both genetic and environmental risk factors. Meta-analyses of genome-wide association studies have identified &gt;150 loci associated with CAD and myocardial infarction susceptibility in humans. A majority of these variants reside in non-coding regions and are co-inherited with hundreds of candidate regulatory variants, presenting a challenge to elucidate their functions. Herein, we use integrative genomic, epigenomic and transcriptomic profiling of perturbed human coronary artery smooth muscle cells and tissues to begin to identify causal regulatory variation and mechanisms responsible for CAD associations. Using these genome-wide maps, we prioritize 64 candidate variants and perform allele-specific binding and expression analyses at seven top candidate loci: 9p21.3, SMAD3, PDGFD, IL6R, BMP1, CCDC97/TGFB1 and LMOD1. We validate our findings in expression quantitative trait loci cohorts, which together reveal new links between CAD associations and regulatory function in the appropriate disease context.</p><dl class="contributors"><dt>Authors: </dt><dd><ul><li class="contributor"><a href="https://www.linkedin.com/in/milos-pjanic-a3b39746">Milos Pjanic</a>, </li><li class="contributor"><a href="https://www.linkedin.com/in/clint-miller-b6b51122">Clint Miller</a>, </li><li class="contributor"><a href="https://www.linkedin.com/in/thomas-quertermous-08bb4024">Thomas Quertermous</a>, </li><li class="contributor">Stephen B. Montgomery, </li><li class="contributor">Eric E. Schadt</li></ul></dd></dl></li><li class="publication"><header><h4 class="item-title"><a href="https://www.linkedin.com/redir/redirect?url=http%3A%2F%2Frdcu%2Ebe%2Fj45f&amp;urlhash=87Fi&amp;trk=prof-publication-title-link" title="Genetics and Genomics of Coronary Artery Disease" class="external-link">Genetics and Genomics of Coronary Artery Disease</a></h4><h5 class="item-subtitle">Current Cardiology Reports</h5><div class="meta"><span class="date-range"><time>September 2016</time></span></div></header><p class="description">Coronary artery disease (or coronary heart disease), is the leading cause of mortality in many of the developing as well as the developed countries of the world. Cholesterol-enriched plaques in the heart’s blood vessels combined with inflammation lead to the lesion expansion, narrowing of blood vessels, reduced blood flow, and may subsequently cause lesion rupture and a heart attack. Even though several environmental risk factors have been established, such as high LDL-cholesterol, diabetes, and high blood pressure, the underlying genetic composition may substantially modify the disease risk; hence, genome composition and gene-environment interactions may be critical for disease progression. Ongoing scientific efforts have seen substantial advancements related to the fields of genetics and genomics, with the major breakthroughs yet to come. As genomics is the most rapidly advancing field in the life sciences, it is important to present a comprehensive overview of current efforts. Here, we present a summary of various genetic and genomics assays and approaches applied to coronary artery disease research.</p><dl class="contributors"><dt>Authors: </dt><dd><ul><li class="contributor"><a href="https://www.linkedin.com/in/milos-pjanic-a3b39746">Milos Pjanic</a>, </li><li class="contributor"><a href="https://www.linkedin.com/in/clint-miller-b6b51122">Clint Miller</a>, </li><li class="contributor"><a href="https://www.linkedin.com/in/robert-wirka-368678bb">Robert Wirka</a>, </li><li class="contributor"><a href="https://www.linkedin.com/in/juyong-brian-kim-787b821">Juyong Brian Kim</a>, </li><li class="contributor"><a href="https://www.linkedin.com/in/daniel-direnzo-8a043a31">Daniel DiRenzo</a>, </li><li class="contributor"><a href="https://www.linkedin.com/in/thomas-quertermous-08bb4024">Thomas Quertermous</a></li></ul></dd></dl></li><li class="publication"><header><h4 class="item-title"><a href="https://www.linkedin.com/redir/redirect?url=https%3A%2F%2Fgithub%2Ecom%2Fmilospjanic%2FmoveExt2MasterTable&amp;urlhash=MGLR&amp;trk=prof-publication-title-link" title="moveExt2MasterTable, RNASeq preprocessing script, output files&gt;mastertable for DESeq/EdgeR,useful conversion subfolders&gt;samples" class="external-link">moveExt2MasterTable, RNASeq preprocessing script, output files&gt;mastertable for DESeq/EdgeR,useful conversion subfolders&gt;samples</a></h4><h5 class="item-subtitle">GitHub</h5><div class="meta"><span class="date-range"><time>August 2016</time></span></div></header><p class="description">This is a useful script if you do RNAseq for multiple samples/conditions and have featureCounts (or HTSeq, Kallisto, etc.) output files located in different subfolders with only the subfolder name distinguishing samples/conditions. <br />Script will go into subfolders, copy the output files, give them the correct sample names, and create a master table for DESeq or EdgeR using the subfolder names and column you want to extract from the files.</p><dl class="contributors"><dt>Authors: </dt><dd><ul><li class="contributor"><a href="https://www.linkedin.com/in/milos-pjanic-a3b39746">Milos Pjanic</a></li></ul></dd></dl></li><li class="publication"><header><h4 class="item-title">Integrative functional genomics identifies regulatory mechanisms at coronary artery disease loci</h4><h5 class="item-subtitle">American Society of Human Genetics 66th Annual Meeting</h5><div class="meta"><span class="date-range"><time>October 2016</time></span></div></header><dl class="contributors"><dt>Authors: </dt><dd><ul><li class="contributor"><a href="https://www.linkedin.com/in/milos-pjanic-a3b39746">Milos Pjanic</a></li></ul></dd></dl></li><li class="publication"><header><h4 class="item-title">Functional fine-mapping of coronary artery disease risk variants</h4><h5 class="item-subtitle">American Society of Human Genetics 66th Annual Meeting</h5><div class="meta"><span class="date-range"><time>October 2016</time></span></div></header><dl class="contributors"><dt>Authors: </dt><dd><ul><li class="contributor"><a href="https://www.linkedin.com/in/milos-pjanic-a3b39746">Milos Pjanic</a></li></ul></dd></dl></li><li class="publication"><header><h4 class="item-title">Interpretable deep learning approaches to understand the genetic and regulatory basis of coronary artery disease</h4><h5 class="item-subtitle">American Society of Human Genetics 66th Annual Meeting</h5><div class="meta"><span class="date-range"><time>October 2016</time></span></div></header><dl class="contributors"><dt>Authors: </dt><dd><ul><li class="contributor"><a href="https://www.linkedin.com/in/milos-pjanic-a3b39746">Milos Pjanic</a></li></ul></dd></dl></li><li class="publication"><header><h4 class="item-title">GWAS candidate gene for coronary artery disease TCF21 interacts with the aryl-hydrocarbon receptor to modify coronary smooth muscle cell response to pro-atherogenic stimuli</h4><h5 class="item-subtitle">American Society of Human Genetics 66th Annual Meeting</h5><div class="meta"><span class="date-range"><time>October 2016</time></span></div></header><dl class="contributors"><dt>Authors: </dt><dd><ul><li class="contributor"><a href="https://www.linkedin.com/in/milos-pjanic-a3b39746">Milos Pjanic</a></li></ul></dd></dl></li><li class="publication"><header><h4 class="item-title"><a href="https://www.linkedin.com/redir/redirect?url=http%3A%2F%2Fcirc%2Eahajournals%2Eorg%2Fcontent%2F134%2FSuppl_1%2FA18374&amp;urlhash=tLsH&amp;trk=prof-publication-title-link" title="TCF21 and Aryl-Hydrocarbon Receptor Gene Cooperate to Activate a Pro-Atherosclerotic Gene Expression Program" class="external-link">TCF21 and Aryl-Hydrocarbon Receptor Gene Cooperate to Activate a Pro-Atherosclerotic Gene Expression Program</a></h4><h5 class="item-subtitle">Circulation</h5><div class="meta"><span class="date-range"><time>November 2016</time></span></div></header><p class="description">Conference: American Heart Association's 2016 Scientific Sessions and Resuscitation Science Symposium. Session Title: Lipid and Lipoprotein Metabolism: Genetics and Genomics, At New Orleans, Louisiana, US, Volume: November 11, 2016 — Volume 134, Issue Suppl 1. Circulation. 2016;134:A18374<br /></p><dl class="contributors"><dt>Authors: </dt><dd><ul><li class="contributor"><a href="https://www.linkedin.com/in/milos-pjanic-a3b39746">Milos Pjanic</a></li></ul></dd></dl></li></ul></section><section id="education" data-section="educationsDetails" class="profile-section"><h3 class="title">Education</h3><ul class="schools"><li class="school"><header><h4 class="item-title">University of Lausanne</h4><h5 class="item-subtitle"><span class="translated translation" data-field-name="Education.DegreeName">Doctorate degree, Life Sciences</span><span class="original translation" data-field-name="Education.DegreeName">Doctorate degree, Life Sciences</span></h5></header><div class="meta"><span class="date-range"><time>2006</time> – <time>2010</time></span></div></li><li class="school"><header><h5 class="logo"><a href="https://www.linkedin.com/edu/university-of-belgrade-11685?trk=ppro_sprof"><img lazyload="true" class="lazy-load" alt="University of Belgrade" data-delayed-url="https://media.licdn.com/mpr/mpr/shrink_100_100/p/3/005/01e/11a/09c3042.png" width="60" height="60" /></a></h5><h4 class="item-title"><a href="https://www.linkedin.com/edu/university-of-belgrade-11685?trk=ppro_sprof">University of Belgrade</a></h4><h5 class="item-subtitle"><span class="translated translation" data-field-name="Education.DegreeName">Master of Science, MSc, Molecular Biology and Physiology</span><span class="original translation" data-field-name="Education.DegreeName">Master of Science, MSc, Molecular Biology and Physiology</span></h5></header><div class="meta"><span class="date-range"><time>1997</time> – <time>2005</time></span></div><div class="description" data-section="educations"><p>Activities and Societies: Major: Genetic Engineering and Biotechnology</p></div></li><li class="school"><header><h4 class="item-title">Fifth Belgrade gymnasium</h4><h5 class="item-subtitle"><span class="translated translation" data-field-name="Education.DegreeName">Section of Natural Sciences and Mathematics</span><span class="original translation" data-field-name="Education.DegreeName">Section of Natural Sciences and Mathematics</span></h5></header><div class="meta"><span class="date-range"><time>1993</time> – <time>1997</time></span></div></li></ul></section><section id="certifications" data-section="certifications" class="profile-section"><h3 class="title">Certifications</h3><ul class="certifications"><li class="certification"><header><h4 class="item-title">R Programming: Part 1</h4><h5 class="item-subtitle">Stanford Medicine Lane Medical Library</h5><div class="meta"><span class="date-range"><time>August 2015</time> – Present</span></div></header></li><li class="certification"><header><h4 class="item-title">R Programming: Part 2</h4><h5 class="item-subtitle">Stanford Medicine Lane Medical Library</h5><div class="meta"><span class="date-range"><time>August 2015</time> – Present</span></div></header></li><li class="certification"><header><h4 class="item-title">Scientific Illustration 2: Creating Figures with Illustrator and Flash</h4><h5 class="item-subtitle">Stanford Medicine Lane Medical Library</h5><div class="meta"><span class="date-range"><time>August 2015</time> – Present</span></div></header></li><li class="certification"><header><h4 class="item-title">Statistical analysis applied to genome and proteome analyses</h4><h5 class="item-subtitle">EMBnet, EPFL | École polytechnique fédérale de Lausanne, Switzerland</h5><div class="meta"><span class="date-range"><time>March 2008</time> – Present</span></div></header></li><li class="certification"><header><h4 class="item-title">Introduction to statistics for biologists</h4><h5 class="item-subtitle">University of Basel, Switzerland</h5><div class="meta"><span class="date-range"><time>March 2007</time> – Present</span></div></header></li><li class="certification"><header><h4 class="item-title">DNA Microarrays data analysis</h4><h5 class="item-subtitle">EMBnet, EPFL | École polytechnique fédérale de Lausanne, Switzerland</h5><div class="meta"><span class="date-range"><time>September 2006</time> – Present</span></div></header></li></ul></section><section id="groups" data-section="groups" class="profile-section"><div class="reg-upsell "><h2>View Milos’ full profile to...</h2><ul><li>See who you know in common</li><li>Get introduced</li><li>Contact <strong>Milos</strong> directly</li></ul><a href="https://www.linkedin.com/start/view-full-profile?_ed=0_3AyDXbdm3z1XO_NPPHT6udWTYe7jO6pEyPuGrmIDyu7xRsoBHEEX3zS0BJbp1Wpk&amp;trk=public_profile_advocate-cta-view-full" class="signup-button">View Milos’ Full Profile</a></div></section></div><div id="aux"><section class="insights"><div id="locale-switcher"><input type="checkbox" id="locale-switcher-expander-state-checkbox" class="expander-checkbox" /><label for="locale-switcher-expander-state-checkbox" id="locale-switcher-expander-state-checkbox" class="dropdown-label">View this profile in another language</label><ul class="locales"><li class="locale current"><a href="https://www.linkedin.com/in/milos-pjanic-a3b39746">English</a></li><li class="locale"><a href="https://www.linkedin.com/in/milos-pjanic-a3b39746/de">German</a></li></ul><script>document.addEventListener('click',function(e){if(e.target.id!=='locale-switcher-expander-state-checkbox'){document.getElementById('locale-switcher-expander-state-checkbox').checked=false;}});</script></div><div class="name-search"><h3 class="title">Search by name</h3><p class="blurb">Over 400 million professionals are already on LinkedIn. Find who you know.</p><form action="https://www.linkedin.com/pub/dir/" method="get"><label for="firstName">First Name</label><input type="text" name="first" id="firstName" placeholder="First Name" /><label for="lastName">Last Name</label><input type="text" name="last" id="lastName" placeholder="Last Name" /><input type="hidden" name="trk" value="prof-samename-search-submit" /><button aria-label="Search" type="submit"><span></span></button><p class="tip"><strong>Example: </strong> <a href="https://www.linkedin.com/in/jeffweiner08">Jeff Weiner</a></p></form></div><section class="badge profile-section"><h3 class="title">Public profile badge</h3><p class="description">Include this LinkedIn profile on other websites</p><a class="cta" href="https://www.linkedin.com/badges/profile/create?vanityname=milos-pjanic-a3b39746&amp;trk=profile-badge-public-profile">View profile badges</a></section><div class="browse-map"><h3 class="title">People Also Viewed</h3><ul><li class="profile-card"><a href="https://www.linkedin.com/in/marie-laurent-771a8066?trk=pub-pbmap"><img lazyload="true" class="lazy-loaded" width="100" height="100" src="https://media.licdn.com/mpr/mpr/shrink_100_100/p/5/000/212/0e8/3960848.jpg" /></a><div class="info"><h4 class="item-title"><a href="https://www.linkedin.com/in/marie-laurent-771a8066?trk=pub-pbmap">Marie Laurent</a></h4><p class="headline">Étudiant(e) (Université de Lausanne)</p></div></li><li class="profile-card"><a href="https://www.linkedin.com/in/dusserre-yves-05aa938?trk=pub-pbmap"><img class="lazy-loaded" width="100" height="100" src="https://static.licdn.com/scds/common/u/images/themes/katy/ghosts/person/ghost_person_100x100_v1.png" /></a><div class="info"><h4 class="item-title"><a href="https://www.linkedin.com/in/dusserre-yves-05aa938?trk=pub-pbmap">Dusserre Yves</a></h4><p class="headline">technician lab manager at UNIL</p></div></li><li class="profile-card"><a href="https://www.linkedin.com/in/amanda-finger-stadler-5825a9a0?trk=pub-pbmap"><img lazyload="true" class="lazy-load" data-delayed-url="https://media.licdn.com/mpr/mpr/shrink_100_100/p/4/005/08d/0e4/3b7ea19.jpg" width="100" height="100" /></a><div class="info"><h4 class="item-title"><a href="https://www.linkedin.com/in/amanda-finger-stadler-5825a9a0?trk=pub-pbmap">Amanda Finger Stadler</a></h4><p class="headline">Postdoctoral Research Fellow at Augusta University</p></div></li><li class="profile-card"><a href="https://www.linkedin.com/in/samuele-ponzio-b95912a3?trk=pub-pbmap"><img lazyload="true" class="lazy-load" data-delayed-url="https://media.licdn.com/mpr/mpr/shrink_100_100/AAEAAQAAAAAAAAJNAAAAJGFjOTQ5OTFiLTE2ODItNGFkMS04YWQ3LWFiMjVjZjg1NzQxOQ.jpg" width="100" height="100" /></a><div class="info"><h4 class="item-title"><a href="https://www.linkedin.com/in/samuele-ponzio-b95912a3?trk=pub-pbmap">Samuele Ponzio</a></h4><p class="headline">Compagnia Finzi Pasca</p></div></li><li class="profile-card"><a href="https://www.linkedin.com/in/paola-benaglio-b3426758?trk=pub-pbmap"><img lazyload="true" class="lazy-load" data-delayed-url="https://media.licdn.com/mpr/mpr/shrink_100_100/AAEAAQAAAAAAAAI-AAAAJDRjYjk3YmJkLWExOTEtNDJlMi1iNzY2LTRjOWNiZDc4NWFiYg.jpg" width="100" height="100" /></a><div class="info"><h4 class="item-title"><a href="https://www.linkedin.com/in/paola-benaglio-b3426758?trk=pub-pbmap">Paola Benaglio</a></h4><p class="headline">Postdoctoral fellow, UCSD</p></div></li><li class="profile-card"><a href="https://www.linkedin.com/in/roman-rudzi%C5%84ski-893b15107?trk=pub-pbmap"><img data-delayed-url="https://static.licdn.com/scds/common/u/images/themes/katy/ghosts/person/ghost_person_100x100_v1.png" class="lazy-load" width="100" height="100" /></a><div class="info"><h4 class="item-title"><a href="https://www.linkedin.com/in/roman-rudzi%C5%84ski-893b15107?trk=pub-pbmap">Roman Rudziński</a></h4><p class="headline">--</p></div></li><li class="profile-card"><a href="https://www.linkedin.com/in/runweizhang?trk=pub-pbmap"><img data-delayed-url="https://static.licdn.com/scds/common/u/images/themes/katy/ghosts/person/ghost_person_100x100_v1.png" class="lazy-load" width="100" height="100" /></a><div class="info"><h4 class="item-title"><a href="https://www.linkedin.com/in/runweizhang?trk=pub-pbmap">Runwei Zhang</a></h4><p class="headline">Quantitative Analyst at Systematica Investments</p></div></li><li class="profile-card"><a href="https://www.linkedin.com/in/tania-binaghi-aa69078b?trk=pub-pbmap"><img data-delayed-url="https://static.licdn.com/scds/common/u/images/themes/katy/ghosts/person/ghost_person_100x100_v1.png" class="lazy-load" width="100" height="100" /></a><div class="info"><h4 class="item-title"><a href="https://www.linkedin.com/in/tania-binaghi-aa69078b?trk=pub-pbmap">Tania Binaghi</a></h4><p class="headline">--</p></div></li><li class="profile-card"><a href="https://www.linkedin.com/in/miroslav-popovic-0092417?trk=pub-pbmap"><img data-delayed-url="https://static.licdn.com/scds/common/u/images/themes/katy/ghosts/person/ghost_person_100x100_v1.png" class="lazy-load" width="100" height="100" /></a><div class="info"><h4 class="item-title"><a href="https://www.linkedin.com/in/miroslav-popovic-0092417?trk=pub-pbmap">Miroslav Popovic</a></h4><p class="headline">Software Engineer at Cisco</p></div></li><li class="profile-card"><a href="https://www.linkedin.com/in/joe-weber-18a29995?trk=pub-pbmap"><img data-delayed-url="https://static.licdn.com/scds/common/u/images/themes/katy/ghosts/person/ghost_person_100x100_v1.png" class="lazy-load" width="100" height="100" /></a><div class="info"><h4 class="item-title"><a href="https://www.linkedin.com/in/joe-weber-18a29995?trk=pub-pbmap">Joe Weber</a></h4><p class="headline">PhD Student at University of Zurich</p></div></li></ul></div></section><section class="jserp"><h3 class="title">Jobs similar to Milos’</h3><ul><li><a href="https://www.linkedin.com/jobs/stanford-university-school-of-medicine-jobs?trk=pprof-jserp">Stanford University School of Medicine jobs</a></li><li><a href="https://www.linkedin.com/jobs/postdoctoral-research-fellow-jobs?trk=pprof-jserp">Postdoctoral Research Fellow jobs</a></li></ul></section><section id="ad" class="profile-section"><iframe width="300" scrolling="no" height="250" frameborder="0" allowtransparency="true" border="0" data-src="/csp/dtag?sz=300x250&amp;ti=2&amp;p=1&amp;z=full_profile&amp;_x=%3Bcompany%3D554298334%3Bpcntry%3Dus" src="/csp/dtag?sz=300x250&amp;ti=2&amp;p=1&amp;z=full_profile&amp;_x=%3Bcompany%3D554298334%3Bpcntry%3Dus"></iframe><script>window.addEventListener('load',function(){var iframes=document.getElementsByTagName("iframe");for(var i=0;i&lt;iframes.length;i++){var iframe=iframes[i],src=iframe.getAttribute('data-src');if(src){iframe.src=src;}}});</script></section><div class="aux-module course-recommendations profile-section"><h3>Add new skills with these courses</h3><ul><li><a class="course-image" href="https://www.linkedin.com/learning/introduction-to-data-science/?trk=publicProfile-public_profile_v3_desktop-learningFeedm014%3Aa001--420305_learning" data-li-trk-code="pprof-sb-lil-photo"><img class="lazy-load" alt="Introduction to Data Science" data-delayed-url="https://cdn.lynda.com/courses/420305-636121264639952837_120x213_thumb.jpg" /></a><h4><a href="https://www.linkedin.com/learning/introduction-to-data-science/?trk=publicProfile-public_profile_v3_desktop-learningFeedm014%3Aa001--420305_learning" data-li-trk-code="prof-sb-lil-name">Introduction to Data Science</a></h4><p>Viewers: 30337</p></li><li><a class="course-image" href="https://www.linkedin.com/learning/spss-statistics-essential-training/?trk=publicProfile-public_profile_v3_desktop-learningFeedm014%3Aa001--182376_learning" data-li-trk-code="pprof-sb-lil-photo"><img class="lazy-load" alt="SPSS Statistics Essential Training" data-delayed-url="https://cdn.lynda.com/courses/182376-635726449434576855_120x212_thumb.jpg" /></a><h4><a href="https://www.linkedin.com/learning/spss-statistics-essential-training/?trk=publicProfile-public_profile_v3_desktop-learningFeedm014%3Aa001--182376_learning" data-li-trk-code="prof-sb-lil-name">SPSS Statistics Essential Training</a></h4><p>Viewers: 37123</p></li><li><a class="course-image" href="https://www.linkedin.com/learning/tableau-9-essential-training/?trk=publicProfile-public_profile_v3_desktop-learningFeedm014%3Aa001--386886_learning" data-li-trk-code="pprof-sb-lil-photo"><img class="lazy-load" alt="Tableau 9 Essential Training" data-delayed-url="https://cdn.lynda.com/courses/386886-636123065496296653_121x215_thumb.jpg" /></a><h4><a href="https://www.linkedin.com/learning/tableau-9-essential-training/?trk=publicProfile-public_profile_v3_desktop-learningFeedm014%3Aa001--386886_learning" data-li-trk-code="prof-sb-lil-name">Tableau 9 Essential Training</a></h4><p>Viewers: 30610</p></li></ul><span class="learning-landing"><a class="label" href="https://www.linkedin.com/learning/?trk=publicProfile_public_profile_v3_desktop_learningFeedm014%3Aa001_viewMoreCourses_learning" data-li-trk-code="pprof-sb-lil-seemore">View more courses</a><span aria-hidden="true" class="arrow-icon"></span></span></div></div><div id="directory" class="us"><h3 class="title">LinkedIn member directory:</h3><ol type="a" class="primary"><li><a href="https://www.linkedin.com/directory/people-a/">a</a></li><li><a href="https://www.linkedin.com/directory/people-b/">b</a></li><li><a href="https://www.linkedin.com/directory/people-c/">c</a></li><li><a href="https://www.linkedin.com/directory/people-d/">d</a></li><li><a href="https://www.linkedin.com/directory/people-e/">e</a></li><li><a href="https://www.linkedin.com/directory/people-f/">f</a></li><li><a href="https://www.linkedin.com/directory/people-g/">g</a></li><li><a href="https://www.linkedin.com/directory/people-h/">h</a></li><li><a href="https://www.linkedin.com/directory/people-i/">i</a></li><li><a href="https://www.linkedin.com/directory/people-j/">j</a></li><li><a href="https://www.linkedin.com/directory/people-k/">k</a></li><li><a href="https://www.linkedin.com/directory/people-l/">l</a></li><li><a href="https://www.linkedin.com/directory/people-m/">m</a></li><li><a href="https://www.linkedin.com/directory/people-n/">n</a></li><li><a href="https://www.linkedin.com/directory/people-o/">o</a></li><li><a href="https://www.linkedin.com/directory/people-p/">p</a></li><li><a href="https://www.linkedin.com/directory/people-q/">q</a></li><li><a href="https://www.linkedin.com/directory/people-r/">r</a></li><li><a href="https://www.linkedin.com/directory/people-s/">s</a></li><li><a href="https://www.linkedin.com/directory/people-t/">t</a></li><li><a href="https://www.linkedin.com/directory/people-u/">u</a></li><li><a href="https://www.linkedin.com/directory/people-v/">v</a></li><li><a href="https://www.linkedin.com/directory/people-w/">w</a></li><li><a href="https://www.linkedin.com/directory/people-x/">x</a></li><li><a href="https://www.linkedin.com/directory/people-y/">y</a></li><li><a href="https://www.linkedin.com/directory/people-z/">z</a></li><li><a href="https://www.linkedin.com/directory/people-1/">more</a></li><li class="country-search">Browse members <a href="https://www.linkedin.com/directory/country_listing/">by country</a></li></ol></div><script type="text/javascript">(function(){var DOMAIN_NAME=window.location.hostname.replace(/.*\.([^\.]+\.[^\.]+)/,'$1'),DOMAIN_NAME_PROD='linkedin.com';function getRemoteFrameUrl(){return('https:'===document.location.protocol?'https://':'http://')+'platform.'+DOMAIN_NAME+'/js/thirdPartyJSDelegatorFrame.html';}function loadExternalTracking(){var thirdPartyJs=new Espany({remote:getRemoteFrameUrl(),ready:function(){if(DOMAIN_NAME!==DOMAIN_NAME_PROD){return;}thirdPartyJs.loadScript('analyticsjs').result(function(msg){thirdPartyJs.run('ga',['create','UA-62256447-1',{'cookieDomain':DOMAIN_NAME_PROD}]);thirdPartyJs.run('ga',['set','referrer',document.referrer]);thirdPartyJs.run('ga',['set','location',window.location.href]);thirdPartyJs.run('ga',['send','pageview',{page:'public_profile_v3_desktop'}]);});}});}window.addEventListener('load',loadExternalTracking);})();</script></div>
</main>
<script type="text/javascript" src="https://static.licdn.com/scds/concat/common/js?h=a06jpss2hf43xwxobn0gl598m-44hhbxag3hinac547ym9vby09-a4lcy9x33w9gvnro4s0fw3e8z-9zz2lhu3eq1epk7sq1t8cdb5s-edgsl2z4e4gk56cy2m5kbpp1q-9o2gces8tdiaq46j2fgjkg6d4-bctwwqj7p01tcj2smshz2bboe-aaykw1861wb5yl2yeseicumeh-bftaa82sjwcbrohoe28skni7b-2r8hqscu9unerft0cqwr58gz1-8ycvggo1571xgrdka3utvcyml-cfabcg4u1cj0em4yissh5mfxu"></script>
<footer id="layout-footer" class="layout-header-or-footer" role="contentinfo">
<div class="wrapper">
<p class="copyright guest"><span>LinkedIn Corporation</span> <em>© 2016</em></p>
<ul class="nav-legal" role="navigation" aria-label="Footer Legal Menu">
<li role="menuitem"><a href="http://www.linkedin.com/legal/user-agreement?trk=hb_ft_userag">User Agreement</a></li>
<li role="menuitem"><a href="http://www.linkedin.com/legal/privacy-policy?trk=hb_ft_priv">Privacy Policy</a></li>
<li role="menuitem"><a href="https://linkedin.com/help/linkedin/answer/34593?lang=en">Community Guidelines</a></li>
<li role="menuitem"><a href="http://www.linkedin.com/legal/cookie-policy?trk=hb_ft_cookie">Cookie Policy</a></li>
<li role="menuitem"><a href="http://www.linkedin.com/legal/copyright-policy?trk=hb_ft_copy">Copyright Policy</a></li>
<li role="menuitem"><a href="/psettings/guest-email-unsubscribe?trk=hb_ft_gunsub" rel="nofollow">Unsubscribe</a></li>
</ul>
</div>
</footer>
<script data-page-js-type="config">(function(n,r,a){r=window[n]=window[n]||{};a=r['WebTracking']=r['WebTracking']||{};a['URLs']={'saveWebActionTrackURL':'\/lite\/secure-web-action-track?csrfToken=ajax%3A4419914781333978430'};}('__li__config_registry__'));</script>
<script data-page-js-type="lix">(function(n,r,a){r=window[n]=window[n]||{};r['global_bsp_notice_type']='warning';r['global_bsp_notice_autoHide']='false';r['global_bsp_view_threshold']='c5';}('__li__lix_registry__'));</script>
<script data-page-js-type="i18n">(function(n,r,a){r=window[n]=window[n]||{};r['global_browser_unsupported_notice']='Looks like you\'re using a browser that\'s not supported. &lt;a target=\"_blank\" href=\"https:\/\/linkedin.com\/help\/linkedin\/answer\/4135?lang=en\"&gt;Learn more about browsers you can use.&lt;\/a&gt;';}('__li__i18n_registry__'));</script>
<script data-page-js-type="config">(function(n,r,a){r=window[n]=window[n]||{};a=r['global:browserSupportPolicy']=r['global:browserSupportPolicy']||{};a['supportedBrowserMinVersions']={ie:'v10',firefox:'v38',opera:'control',safari:'v6.1',chrome:'v42',mobileSafari:'v7',android:'v2.3',androidChrome:'v0'};}('__li__config_registry__'));</script>
<script type="text/javascript">(function(d,f){var modQuot=parseInt(f.substring(1),10);var now=(+new Date());if(typeof modQuot==='number'&amp;&amp;(now%modQuot!==0)){return;}
function go(){var a=d.createElement('iframe');a.style.display='none';a.setAttribute('sandbox','allow-scripts allow-same-origin');a.src='//platform.linkedin.com/js/thirdPartyJSDelegatorFrame.html?type=dyndns';if(d.body){d.body.appendChild(a);}}
if(window.addEventListener){window.addEventListener('load',go,false);}else if(window.addEvent){window.addEvent('onload',go);}}(document,'n10'));</script>
<script type="text/javascript" src="https://static.licdn.com/scds/concat/common/js?h=3kp2aedn5pmamdr4dk4n8atur-3ti5bgrnb6idjtk0w4chaigxe-5hqr1i1uoezoj0z1s5gcxojf2-71o37tcjwl0ishto9izvyml3i-3bbdjshpw5ov0rwa8xe08tp97-cayct4cirf7n0f9z1xsg84g0q-dktkawxk7k8pixuh5g8z5ku32-213zbp2wzp99lviwl8g2cvq6i-1lknwtftishpdmobzm413yc7u-bcxa0v9ke411pjpmz4s239f9b-10wg3j2jlwnawjalr4lur4ho3-82rcsw42m1wbgsti4m3j0kvg6-f3la2n4kbk7vr56j54qax1oif-1eq1il9757v2zkuru6hu14q2e-8sox1gztdjnz2un89fi8fyw35-8hdbl769kuhp0h4bsexhsbks0-d8q8zy9bxe530nqv2fct70nwo-c6ct0moql4p4ngtzltmf8l3ly-8h514j3fiwnzuwkt66sbxsu8f-di2z9sra5co9la7ogqyesywin"></script>
<script src="https://static.licdn.com/sc/h/8pm7pmdzmzf25w9hj2eq6u9eo,bfujzpfzk9bvfrwcnbea0qswy,4oqtpjlxo1bbna8l5sudgzqun,e8grwlvaswvn8ieubdd40fupc,e4ryxmbtlb3ketn2c2mix5s0a" async=""></script>
<script type="text/javascript">if(!window.LI){window.LI={};}
LI.RUM=LI.RUM||{};(function(RUM,win){var doc=win.document;RUM.flags=RUM.flags||{};RUM.flags['host-flag']="control";RUM.flags['pop_beacons_frequency']="n10-ap0-va0-tx0-sg0-db0-hk0-sp0-ln0-ch0-sy0-mu0-mi0";RUM.flags['rs_timings_individual']="n1000";RUM.flags['rs_timings_individual_detail']="enabled";RUM.urls=RUM.urls||{};RUM.urls['rum-track']="\/lite\/rum-track?csrfToken=ajax%3A4419914781333978430";RUM.urls['boomerang-bw-img']="https:\/\/static.licdn.com\/scds\/common\/u\/lib\/boomerang\/0.9.edge.4ab208445a\/img\/";RUM.base_urls=RUM.base_urls||{};RUM.base_urls['permanent_content']="https:\/\/static.licdn.com\/scds\/common\/u\/";RUM.base_urls['versioned_content']="https:\/\/static.licdn.com\/scds\/concat\/common\/";RUM.base_urls['media_proxy']="https:\/\/media.licdn.com\/media-proxy\/";RUM.serverStartTime=1.480728292249E12;RUM.enabled=true;function getRumScript(){var node=doc.body||doc.head||doc.getElementsByTagName('head')[0],script=doc.createElement('script');script.src=["https://static.licdn.com/scds/concat/common/js?h=ed29nkjpsa16bhrjq4na16owq-1mucgfycc664m7vmhpjgqse65-1l5rurej3h44qodo5rn0cdvyn-8om6v2ckrxsbnwf40t9ta8a7e-8jlhg6lqacthgadello7fgxzm-28w7d5j2k2jtil9ncckolke4m-9jzlwicvu376y9q4vjq77y5ks-1m0whdrwis44c1hoa9mrwhlt4-1uvutm1mpyov7rqhtcf8fksby-aac54ic1fmca5xz1yvc5t9nfe-1hn40w0bomeivihj9lopp4hp2-c0121povror81d0xao0yez4gy"][0];node.appendChild(script);}
if(win.addEventListener){win.addEventListener('load',getRumScript);}
else{win.attachEvent('onload',getRumScript);}}(LI.RUM,window));</script>
<meta name="detectAdBlock" content="//platform.linkedin.com/js/px.js" />
<script src="https://static.licdn.com/scds/concat/common/js?h=69w33ou4umkyupw2uqgn7za7w" async="" defer=""></script>
<script class="comscore-tracking">(function(_e,_r){var isNielsenDisabled=true;var providers={'COMSCORE':{'treatment':'control'}};if(!isNielsenDisabled){providers['NIELSEN']={'treatment':'control'};}
var fireComscore=function(){var comScore=window.COMSCORE;if(comScore){comScore.beacon({c1:2,c2:6402952,c3:'',c4:'',c5:'',c6:'',c15:''});}};var uc=window.encodeURIComponent,timeStamp=(new Date()).getTime();var fireExternalTracking=function(et){if(typeof et.trackPageChromeInit==='function'){et.trackPageChromeInit(providers);}else{et.setTreatment('enabled_1.0');et.trackWithComScoreForChromeInit();if(!isNielsenDisabled){var img=new Image(1,1);img.onerror=img.onload=function(){img.onerror=img.onload=null;};img.src=["//secure-gl.imrworldwide.com/cgi-bin/m?ci=au-linkedin&amp;cc=1&amp;si=",uc(window.location.href),"&amp;rp=",uc(document.referrer),"&amp;ts=compact&amp;rnd=",timeStamp].join('');}}};var track=function(){var et;if(_e){fireExternalTracking(_e);}else if(_r&amp;&amp;typeof _r.ensure==='function'){try{_r.ensure(['externalTracking'],function(require){try{et=require('externalTracking');fireExternalTracking(et);}catch(e){fireComscore();}});}catch(e){fireComscore();}}else if(_r&amp;&amp;_r._is_li){try{et=require('externalTracking');fireExternalTracking(et);}catch(e){fireComscore();}}else{fireComscore();}};window.addEventListener('load',track);}(window.externalTracking,window.require));</script>
<noscript>
&lt;img src="https://sb.scorecardresearch.com/b?c1=2&amp;amp;c2=6402952&amp;amp;c3=&amp;amp;c4=&amp;amp;c5=&amp;amp;c6=&amp;amp;c15=&amp;amp;cv=1.3&amp;amp;cj=1" style="display:none" width="0" height="0" alt=""/&gt;
</noscript>
<script type="text/javascript">(function(d){function go(){var a=d.createElement('iframe');a.style.display='none';a.setAttribute('sandbox','allow-scripts allow-same-origin');a.src='//radar.cedexis.com/1/11326/radar/radar.html';if(d.body){d.body.appendChild(a);}}
if(window.addEventListener){window.addEventListener('load',go,false);}else if(window.addEvent){window.addEvent('onload',go);}}(document));</script>
</div>


<iframe id="e_1480728292727" name="e_1480728292727" src="https://platform.linkedin.com/js/thirdPartyJSDelegatorFrame.html?e_origin=https://www.linkedin.com&amp;e_channel=e_1480728292727" style="visibility: hidden; width: 1px; height: 1px; position: absolute; left: -999px; top: 0px;"></iframe><script src="https://static.licdn.com/scds/concat/common/js?h=ed29nkjpsa16bhrjq4na16owq-1mucgfycc664m7vmhpjgqse65-1l5rurej3h44qodo5rn0cdvyn-8om6v2ckrxsbnwf40t9ta8a7e-8jlhg6lqacthgadello7fgxzm-28w7d5j2k2jtil9ncckolke4m-9jzlwicvu376y9q4vjq77y5ks-1m0whdrwis44c1hoa9mrwhlt4-1uvutm1mpyov7rqhtcf8fksby-aac54ic1fmca5xz1yvc5t9nfe-1hn40w0bomeivihj9lopp4hp2-c0121povror81d0xao0yez4gy"></script><iframe sandbox="allow-scripts allow-same-origin" src="//radar.cedexis.com/1/11326/radar/radar.html" style="display: none;"></iframe><script src="//platform.linkedin.com/js/px.js?ch=1"></script><script src="//platform.linkedin.com/js/px.js?ch=2"></script></body></html>
https://www.linkedin.com/in/milos-pjanic-a3b39746